src/Mailcode/Commands/Command/ShowVariable.php

```diff
@@ -75,21 +80,44 @@ protected function validateSyntax_no_other_tokens() : void
      * @return Mailcode_Parser_Statement_Tokenizer_Token[]
      * @throws Mailcode_Exception
      */
-    protected function resolveActiveTokens() : array
+    protected function resolveActiveTokens(): array
     {
         $allowed = array($this->getVariableToken());
 
         $encodings = $this->getSupportedEncodings();
 
-        foreach($encodings as $keyword)
-        {
+        foreach ($encodings as $keyword) {
             $token = $this->getEncodingToken($keyword);
-            if($token)
-            {
+            if ($token) {
                 $allowed[] = $token;
+
+                $parameter = $this->requireParams()->getInfo()->getTokenForKeyWord($token->getKeyword());
```

Comment:
This only makes sense if the keyword also supports a parameter, which so far has not been formalized.
It's also not clear whether this can cause trouble for optional parameters, in case keywords are mixed.